validator/testdata/amp4email_feature_tests/amp-autocomplete.out

```diff
@@ -0,0 +1,254 @@
+FAIL
+|  <!--
+|    Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+|
+|    Licensed under the Apache License, Version 2.0 (the "License");
+|    you may not use this file except in compliance with the License.
+|    You may obtain a copy of the License at
+|
+|        http://www.apache.org/licenses/LICENSE-2.0
+|
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests the additional restrictions AMP4EMAIL sets on AMP-AUTOCOMPLETE.
+|  -->
+|  <!DOCTYPE html>
+|  <!-- "enable-amp-autocomplete" is disallowed by the validator -->
+|  <html ⚡4email enable-amp-autocomplete>
+>> ^~~~~~~~~
+amp4email_feature_tests/amp-autocomplete.html:22:0 The attribute 'enable-amp-autocomplete' may not appear in tag 'html'. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#required-markup)
```

Comment:
This is being flagged as disallowed. Is that intentional?

Reply:
Yes -- left a comment in L22 above: `<!-- "enable-amp-autocomplete" is disallowed by the validator -->` Though this has been renamed in the recent commit to be consistent with the PR description.